guides/v2.2/frontend-dev-guide/layouts/xml-instructions.md

```diff
@@ -514,7 +514,7 @@ $cssClass = $this->hasCssClass() ? ' ' . $this->getCssClass() : '';
 ...
 {%endhighlight %}
 
-### &lt;arguments&gt; {#arguments}
```

Comment:
I would not eliminate the "<" and the ">" in this title, it references a node inside layout.

Reply:
@osrecio Could you please check https://github.com/magento/devdocs/pull/2413#issuecomment-403805452 or do you have other suggestion to fix this issue?